functions.yml

```diff
@@ -335,6 +351,7 @@ quality_control:
           doc: The reference allele
```

Comment:
For two lines above: `The array of genotype structs` -> `An array of genotype structs with a `calls` field`

Reply:
Done